js/models/listing/Listing.js

```diff
@@ -288,6 +288,7 @@ export default class extends BaseModel {
   }
 
   parse(response) {
+    this.unparsedResponse = JSON.parse(JSON.stringify(response)); // deep clone
```

Comment:
It's not the same. Parse uses the currency util function to attempt to convert the amounts from base units (e.g. satoshi) to non base units or, if fiat, integer to decimal.

But, if the currency is not recognized, then the the currency util will return undefined. So... if we ever need to get those amounts (I think there's only one use case so far), then it would need to come form the unparsed response.